.gitignore

```diff
@@ -30,3 +30,6 @@ notes.md
 tmp/
 
 public/frontend-config.json
+
+# Sentry Config File
+.env.sentry-build-plugin
```

Comment:
Needed? I don't see the file even after a build

Reply:
Is needed, is done when using the dev run. Was also automatically added by Sentry. Will leave it in.